tests/integration/delete-before-replace/delete_before_replace_test.go

```diff
@@ -123,57 +135,6 @@ func TestPod(t *testing.T) {
 					assert.Equal(t, "nginx:1.15-alpine", image)
 				},
 			},
-			{
-				Dir:      "step3",
-				Additive: true,
-				ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
-					assert.NotNil(t, stackInfo.Deployment)
-					assert.Equal(t, 3, len(stackInfo.Deployment.Resources))
-
-					tests.SortResourcesByURN(stackInfo)
-
-					stackRes := stackInfo.Deployment.Resources[2]
-					assert.Equal(t, resource.RootStackType, stackRes.URN.Type())
-
-					provRes := stackInfo.Deployment.Resources[1]
-					assert.True(t, providers.IsProviderType(provRes.URN.Type()))
-
-					//
-					// Assert new Pod is deleted before being replaced with the new Pod, running
-					// nginx:1.13-alpine, EVEN WHEN we change the namespace from "" -> "default". This
```

Comment:
Ok, so I think we want to test that:

1. Changing the `Pod` `""` -> `"default"` DOES NOT result in a replacement.
1. Changing a `Pod` spec in a way that causes a replace, _and also_ changing the namespace from `""` -> `"default"` results in a replacement.

I'm not picky about how we do this, but if we remove this, we probably want to add both of these cases to the `diff_tests.go`, or something.